Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -42,8 +42,6 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
         }
     }
     
-    var didChangeAuthorizationIsFirstCalled = true
-    
```

Comment:
The delegate method `locationManagerDidChangeAuthorization(_:)` is called at startup and whenever the user changes authorization permissions. `didChangeAuthorizationIsFirstCalled` was to deal with the first case. Without this property, the halo view and enable precise location banner would show up when they weren't supposed to.

Reply:
Thanks for flagging that. I've tried on several tests and do not see any issues, at the same time to bring back your implementation from `main` we'll have to possibly bring back `MBNavigationAccuracyAuthorization` and maybe other APIs. I'm inclined to merge this PR as is and create another ticket in scope of which we'll perform more through testing and add fixes if needed.